src/module/tool/how_to_cook.py

```
@@ -1,35 +1,49 @@
 
 _version = "1.5.0"
 
+how_to_cook = on_command("来道菜", aliases={"做菜", "菜", "饿了", "我饿了"}, priority=100, rule=to_me(), block=True)
 
-@command(tokens=['来道菜', '做菜', '菜', '饿了', '我饿了'])
-def reply_how_to_cook(message: RobotMessage):
-    message.reply("正在翻菜谱，请稍等")
-    dishes = load_dishes()
 
+@how_to_cook.handle()
+async def reply_how_to_cook(event: Event, message: Message = CommandArg()):
```

Ruff (0.12.2)

Do not perform function call `CommandArg` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)